### ISSUE TITLE
PANTHER (or reference proteome) is using incorrect AspGD identifiers

### ISSUE THREAD

**Issue by selewis** [geneontology/go-site#93]
The IDs that PANTHER provides for Aspergillus are of this form
AspGD:ANIA_00807

This should in fact by identified as this to be correct
AspGD:ASPL0000059499

**Comment by cmungall** [comment]
Assigned to @mugitty for now (other USC folks may need to be added to group)

**Comment by pgaudet** [comment]
This is still a problem: 

Panther annotations refer to the Uniprot entry: 
http://amigo.geneontology.org/amigo/gene_product/UniProtKB:P50867

But this is the AspDB identifier for the same entry: 
http://amigo.geneontology.org/amigo/gene_product/AspGD:ASPL0000013431

This is a problem as it doubles the number of entries for a species.

@dustine32 Can you please have a look ?

**Comment by dustine32** [comment]
@pgaudet Will take a look. I noticed the other day that our build scripts aren't finding `AspGD` in RefProt's idmapping files. But I probably just need to dig harder.

**Comment by huaiyumi** [comment]
AspGD IDs are not used in PANTHER. Searching ASPL0000013431 in UniProt didn't yield any results. I am sure it is not in Reference Proteome either. So the ID issue mentioned in this ticket is not an issue anymore.
By the way, we use Emericello nidulans only, so there is no issue of doubling the number.

**Comment by dustine32** [comment]
Yep, I guess this one was easy to "debug" but now requires some upstream work. The example protein `UniProtKB:P50867` is from PANTHER15.0 long ID `EMENI|EnsemblGenome=AN8057.2|UniProtKB=P50867`, with species `EMENI` (Emericella nidulans) having taxon ID 227321.

Checking all distinct DB sources in Reference Proteome's `UP000000560_227321.idmapping` file for EMENI, I don't find any `AspGD` prefix available, nor anything else close to that. I can't even find ASPL0000013431 as @huaiyumi pointed out.

My current guess is that the AspGD MOD folks need to either switch to UniProtKB ID's to match PANTHER or work with Reference Proteome to get their identifiers back into the `UP000000560_227321.idmapping` files.

**Comment by dustine32** [comment]
I noticed that AspGD's site does link [ASPL0000013431](http://www.aspergillusgenome.org/cgi-bin/locus.pl?dbid=ASPL0000013431) to [P50867](https://www.uniprot.org/uniprot/P50867/), but the UniProt page does not link back.

@pgaudet Do we have a contact at AspGD? Maybe they'll know what's up.

**Comment by kltm** [comment]
Until @pgaudet can get you something else, you can start with https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv . There is an aspgd entry there. I have an fairly recent email address as well, if you PM me.

**Comment by huaiyumi** [comment]
We should inform Reference Proteome and UniProt and let them resolve it.

**Comment by pgaudet** [comment]
Emailed Maria Matin for Reference Proteomes.
